### ISSUE TITLE
I think you have memory leak

### ISSUE THREAD

**Issue by re0ah** [summerCol/HH-Music#1]
In file bottomwidget.cpp, line 8 you write:

`auto hLayout = new QHBoxLayout(this);`

You allocate memory and don't free it after. I'm not very good at Qt and maybe that's right, just want to report about it. I think the best solution would use smart pointers and write like this:

`unique_ptr<QHBoxLayout> hLayout(new QHBoxLayout(this));`

Or just use stack memory, if it possible. I don't see here are reasons to use dynamic memory.

`QHBoxLayout hLayout = QHBoxLayout(this);`

Good day.

**Comment by summerCol** [comment]
yes  @re0ah , you are right. I consulted the documents of Qt, `QWidget` will auto delete all this widget's children, but  `QHBoxLayout` doesn't inherited from `QWidget`， so it will not be deleted. I will fix it.
